packages/discord.js/src/util/Options.js

```diff
@@ -70,6 +71,14 @@ const { toSnakeCase } = require('./Transformers');
  * Contains various utilities for client options.
  */
 class Options extends null {
+  /**
+   * The default user agent appendix.
+   * @type {string}
+   * @memberof Options
+   * @private
+   */
+  static userAgentAppendix = `discord.js/${version} ${DefaultUserAgentAppendix}`.trim();
```

Comment:
With the changes to `DefaultUserAgentAppendix`, there are no extra spaces here anymore
```suggestion
  static userAgentAppendix = `discord.js/${version} ${DefaultUserAgentAppendix}`;
```

Reply:
oops, yeah. I mistook it for `DefaultUserAgent`. `trimEnd()` can be used here instead then